frontend/src/components/StockData/StockData.jsx

```diff
@@ -87,7 +83,6 @@ export default function StockData(props) {
         <div className="stock-data">
             {isLoading ? (
                 <div className="loading">
-                    <div className="ball"></div>
```

Comment:
No spinner?

Reply:
That's OK. It's not too important to the project. You can add it as a stretch stretch goal at the bottom of the backlog